src/PSPackageProject.psm1

```diff
@@ -11,4 +11,100 @@ function Invoke-PSPackageProjectTest
     )
 
     ## TODO implement calling tests
+    Invoke-BinSkim
+}
+
+function Invoke-BinSkim
+{
+    [CmdletBinding(DefaultParameterSetName='default')]
+    param(
+        [Parameter(ParameterSetName='byPath',Mandatory)]
+        [string]
+        $Path,
+        [Parameter(ParameterSetName='byPath')]
+        [string]
+        $Filter = '*'
+    )
+
+    $sourceName = 'Nuget'
+    Register-PackageSource -ProviderName NuGet -Name $sourceName -Location https://api.nuget.org/v3/index.json -erroraction ignore
+    $packageName = 'microsoft.codeanalysis.binskim'
+    $packageLocation = Join-Path -Path ([System.io.path]::GetTempPath()) -ChildPath 'pspackageproject-packages'
+    Write-Verbose "Finding binskim..." -Verbose
+    $packageInfo = Find-Package -Name $packageName -Source $sourceName
+    if($IsLinux)
+    {
+        $binaryName ='BinSkim'
+        $rid = 'linux-x64'
+    }
+    elseif($IsWindows)
```

Comment:
Should this work in PS 5.1 (`$IsWindows -ne $false`)?

Reply:
```suggestion
    elseif($IsWindows -ne $false)
```